src/components/resultsSection/resultsSection.jsx

```diff
@@ -56,12 +40,16 @@ const ResultsSection = ({
 };
 
 ResultsSection.propTypes = {
-  genres: arrayOf(genreType),
   movies: arrayOf(movieType),
-  modalValues: modalValues,
-  defaultModalValues: modalValues,
-  onModalValuesChange: func,
-  onMovieUpdate: func
+  onMoviesByParamsGet: func
 };
 
-export { ResultsSection };
+const mapStateToProps = (state) => ({
+  movies: state.movies.movieList
```

Comment:
you can get here only length of array as a prop as you use only length in this component

Reply:
good point!